app/serializers/api/v1/participant_serializer.rb

```diff
@@ -9,9 +9,17 @@ class ParticipantSerializer
       include JSONAPI::Serializer::Instrumentation
 
       class << self
+        def induction_record(profile, lead_provider)
+          if profile.active?(lead_provider)
+            profile.current_induction_record
+          else
+            profile.induction_records.joins(:lead_provider).where(partnership: { lead_provider: }).latest
```

Comment:
just a note that max pagination is set to 3000. if a provider uses that, this one api request will fire 3000 sql calls, one on each profile for induction records.